src/js/App/Sidenav/SectionNav.js

```diff
@@ -4,10 +4,12 @@ import { NavGroup } from '@patternfly/react-core/dist/js/components/Nav/NavGroup
 import ExpandableNav from './ExpandableNav';
 import './SectionNav.scss';
 
-const SectionNav = ({ items, title, id, onClick, ...props }) => {
+const sectionTitleMapper = (id) => id.toUpperCase();
```

Comment:
This will make the entire id upperCase, I'd imagine we have a mapper with something like
```JS
const sectionTitleMapper = (id) => ({
  insights: 'Insights'
})?.[id] || '';
```